docker/README.md

```diff
@@ -0,0 +1,27 @@
+# Docker for itwêwina
+
+The current setup was created with the intent of using it in production,
+not for development.
+
+  - Run `sqlite3 db.sqlite3` to create an empty database file
+
+    Otherwise `docker-compose` will create a *directory* instead
```

Comment:
wat

Reply:
When docker is mounting host paths, those paths can be either directories or files. Docker will create the path if it does not exist. Creating an empty directory makes a bit more sense than creating a zero-byte file. If docker were being designed anew, a trailing slash could be required to indicate a directory, but there are already a ton of things out there assuming that specifying a non-existent path without a trailing slash will result in directory creation.